heuristic/classes/Item.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+
+class Item:
+    volume: float
+    customer: int
+
+    @classmethod
+    def create(cls, volume, customer) -> Item:
+        """
+        Creates an item with a set volume and destination (customer). A pickup
+        will have destination 0 (a.k.a. the depot).
+        """
+        item = cls()
+        item.volume = volume
+        item.customer = customer
+
+        return item
+
+    def get_volume(self):
```

Comment:
This is not really Pythonic. Either use a property (```@property```), or just directly access the member variable. I do the latter - the program is not going to be that large, so I'm not too worried about how this breaks encapsulation.